README.md

```diff
@@ -9,103 +9,114 @@ This application is a denormalised warehouse for multiple LIMS.
 2. Reset the database
-```
-    bundle exec rake db:reset
-```
+
+   ```bash
+   bundle exec rake db:reset
+   ```
 
 3. Create the dependent views
-```
-    bundle exec rake db:views:schema:load
-```
+
+   ```bash
+   bundle exec rake db:views:schema:load
+   ```
 
 These actions can also be performed automatically if you run the Docker container of the service
 and pass the environment variables:
-```
+
+```bash
 RAILS_ENV="test"
 INTEGRATION_TEST_SETUP="true"
 ```
 
-
 #### Running the integration tests
-4. Run the integration tests:
-```
-   bundle exec rspec --tag integration
-```
-
-### Preparing to run locally with Traction Service
-
-RabbitMQ is essential for this process, so if you haven't already, install it using:
 
-    brew install rabbitmq
-    brew services start rabbitmq
+1. Run the integration tests:
 
-You can now view the instance running at [http://localhost:15672/](http://localhost:15672/).
+   ```bash
+   bundle exec rspec --tag integration
+   ```
 
-Ensure that you update the `config/environments/development.rb` file in this repository with the exchange name and queue name that are in the `bunny.yml` file for Traction Service.
-The keys to modify here are `config.amqp.main.queue` and `config.amqp.main.exchange`.
-When the unified warehouse is first run, it will look for an exchange and queue of these names and create them if they don't exist.
-However, it is also note worthy that the queue is not set up correctly to work with Traction Service since the queue is not set to `durable = true` and has additional header keys for `x-dead-letter-exchange` and `x-message-ttl` which Traction Service isn't expecting.
-To fix this replace the line in `lib/postman/channel.rb` that reads:
+### Execution
 
-    channel.queue(@queue_name, arguments: queue_arguments)
+Execute the worker to pick up messages in the queue and process them into the
+database:
 
-so that it instead reads:
+```bash
+bundle exec warren consumer start
+```
 
-    channel.queue(@queue_name, durable: true)
+The consumer will run in the foreground, logging to the console. You can stop ti with Ctrl-C.
 
-This will ensure the queue that is generated is compatible with the expectations of Traction Service.
+### Preparing to run locally with Traction Service
 
-### Execution
+RabbitMQ is essential for this process, so if you haven't already, install it using:
 
-Execute the worker to pick up messages in the queue and process them into the
-database:
+```bash
+brew install rabbitmq
+brew services start rabbitmq
+```
 
-    ```bundle exec ./bin/amqp_client start```
+You can now view the instance running at [http://localhost:15672/](http://localhost:15672/).
 
-where `start` instructs it to start. You can also stop a worker by calling `stop`
-or restart it with `restart`.
+You may wish to start the warren consumers with:
 
-To run in non-daemonized mode, which can be useful for debugging:
+```bash
+bundle exec warren consumers start --path='config/warren_traction_dev.yml'
```

Comment:
That's probably better handled by the warren documentation. (Otherwise we'll need to keep it in sync in lots of different places - warren, Sequencescape, unified warehouse, event warehouse....)

Reply:
Actually, I'll pop a reference to warren, plus mentioning `warren help`